src/components/Microsoft.DotNet.UpgradeAssistant/Checks/WcfServerCheck.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Checks
```

Comment:
This may warrant a WCF extension. Not a blocker for sure, but thinking outloud

Reply:
#624